library/alloc/src/vec/mod.rs

```diff
@@ -2392,26 +2392,31 @@ impl<T: Clone, A: Allocator> Vec<T, A> {
         self.spec_extend(other.iter())
     }
 
-    /// Copies elements from `src` range to the end of the vector.
+    /// Appends elements specified by the `src` parameter to the end of the vector.
+    /// The `src` parameter is of type [`RangeBounds`], which means that
```

Comment:
`RangeBounds` is a trait here, not a type. Not exactly sure what the best way to specify this is, though.

Reply:
Since `RangeBounds` is in the signature anyway, and the examples show use of ranges, could this just be:
> Copies elements from one part of the vector to the end.
>
> `src` is the range that the elements will be copied from.

This would be consistent with the wording used in [`slice::copy_within`](https://doc.rust-lang.org/std/primitive.slice.html#method.copy_within)